### PR TITLE
USHIFT-4746: Describe the resource after failed `Oc Wait`

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

File: test/resources/oc.resource [hunk 1]
```diff
@@ -61,10 +61,18 @@ Oc Exec
     RETURN    ${output}
 
 Oc Wait
-    [Documentation]    Run 'oc wait' on a specific resource in the current test namespace
+    [Documentation]    Run 'oc wait' on a specific resource in the current test namespace.
```
Comment (pacevedom) [inline test/resources/oc.resource:64]: I would note here that resource must include the namespace. Or make the namespace a separate argument.

Comment (pmtk) [reply test/resources/oc.resource]: Reworded the comment.

**Review by pacevedom — COMMENTED** [review]
/lgtm

**Pull request merged**